cfg/admin.txt

```diff
@@ -1,2 +1,23 @@
-jmwturner role=admin
-deitylink role=admin
+unid role=admin
+abi79 role=admin
+arcalane role=admin
+asanadas role=admin
+bobbehluvspropane role=admin
```

Comment:
This file didn't exist until https://github.com/vgstation-coders/vgstation13/commit/211a1db8326bd16bcbbf394f9c4a3f0e3dae1eb3 and actually isn't in .gitignore